exp/constraints.lisp

```diff
@@ -24,10 +24,6 @@
 ;; and both shorthands (flag_sum_perspective) and (flag_sum_macro)
 ;; are de facto binary
 
-(defconstraint   wcp-flag-vanishes-outside-of-preprocessing-rows ()
-                 (if-zero    PRPRC
-                             (vanishes!    preprocessing/WCP_FLAG)))
-
```

Comment:
Wrong 'fix': the WCP_FLAG is part of the preprocessing perspective. Imposing it vanish outisde of that perspective makes no sense.